src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs,
+  selectedTabId,
+  setSelectedTabId,
+  onTabSelected }) => {
+  const click = (tabId, selectedId, tabsList) => {
+    if (tabId !== selectedId) {
+      onTabSelected(tabsList.find(tab => tabId === tab.id));
+    }
+  };
+
+  const getIsActive = (tabsList, tabId, selectedId) => {
+    if (tabId === selectedId
+        || tabsList.find(tab => tab.id === selectedId) === undefined) {
+      return 'is-active';
```

Comment:
```suggestion
    if (tabId === selectedId) {
      return 'is-active';
```

looks redundant

Reply:
We need this to set `className='is-active'` to first `<li>` when `selectedTabId` is incorrect, but this code contains an error. I improve it a little bit, so now it works as expected